src/allowed_args.cpp

```diff
@@ -461,7 +461,8 @@ static void addDebuggingOptions(AllowedArgs &allowedArgs, HelpMessageMode mode)
 {
     std::string debugCategories = "addrman, bench, blk, bloom, coindb, db, estimatefee, evict, http, lck, "
                                   "libevent, mempool, mempoolrej, miner, net, parallel, partitioncheck, "
-                                  "proxy, prune, rand, reindex, req, rpc, selectcoins, thin, tor, wallet, zmq";
+                                  "proxy, prune, rand, reindex, req, rpc, selectcoins, thin, tor, wallet, zmq"
```

Comment:
yup, should be:
```
"proxy, prune, rand, reindex, req, rpc, selectcoins, thin, tor, wallet, zmq, "
"graphene"
```

It would seem the debug category at the end is now `"zmqgraphene"`